src/plugins/data_table/server_side_data_table.js

```diff
@@ -137,65 +137,22 @@ var ServerSideDataTable = (function() {
       this._fetchCSV( sUrl, aoData );
     },
 
-    _fetchCSV : function (sUrl, aoData) {
+    _fetchCSV : function (sUrl) {
       if (this.serverSideFiltering) {
         var filterJson = {};
         filterJson.name = "ext_filter_json";
         filterJson.value = this._getFilteringSettings();
-        aoData.push(filterJson);
+        this._goToWindowLocation(sUrl + "&ext_filter_json=" + encodeURIComponent(filterJson.value));
       }
-      // window.location = sUrl;
-      // We want to be able to just set window.location, but we can't yet because backdraft doesn't do anything
-      // with url parameters yet and we lack some infrastructure for performing this operation.
-      // For now we'll use a ajax post with a custom built object url as implemented below.
-      $.ajax({
-        url: sUrl,
-        data : aoData,
-        dataType : "text",
-        cache : false,
-        type : this.ajaxMethod || "GET",
-        success: function(response, status, xhr) {
-          // check for a filename
-          var filename = "";
-          var disposition = xhr.getResponseHeader('Content-Disposition');
-          if (disposition && disposition.indexOf('attachment') !== -1) {
-            var filenameRegex = /filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/;
-            var matches = filenameRegex.exec(disposition);
-            if (matches != null && matches[1]) filename = matches[1].replace(/['"]/g, '');
-          }
-
-          var type = xhr.getResponseHeader('Content-Type');
-          var blob = new Blob([response], { type: type });
-
-          if (typeof window.navigator.msSaveBlob !== 'undefined') {
-            // IE workaround for "HTML7007: One or more blob URLs were revoked by closing
-            // the blob for which they were created. These URLs will no longer resolve as
-            // the data backing the URL has been freed."
-            window.navigator.msSaveBlob(blob, filename);
-          } else {
-            var URL = window.URL || window.webkitURL;
-            var downloadUrl = URL.createObjectURL(blob);
-
-            if (filename) {
-              // use HTML5 a[download] attribute to specify filename
-              var a = document.createElement("a");
-              // safari doesn't support this yet
-              if (typeof a.download === 'undefined') {
-                window.location = downloadUrl;
-              } else {
-                a.href = downloadUrl;
-                a.download = filename;
-                document.body.appendChild(a);
-                a.click();
-              }
-            } else {
-              window.location = downloadUrl;
-            }
+      else {
+        throw new Error("serverSideFiltering is expected to be enabled when _fetchCSV is called");
+      }
+    },
 
-            setTimeout(function () { URL.revokeObjectURL(downloadUrl); }, 100); // cleanup
-          }
-        }
-      })
+    _goToWindowLocation : function(url) {
```

Comment:
Also, Nick, if you're watching this, we're wrapping the window.location inside a method so we can mock/spy on it in Jasmine.